src/ui/patch_hub/latest_patchsets_from_mailing_list.sh

```diff
@@ -1,24 +1,58 @@
 include "${KW_LIB_DIR}/ui/patch_hub/patch_hub_core.sh"
 
+declare -g PAGE=1
+declare -ga formatted_patchsets_list
+
 # This function displays a list of the latest patchsets from a target mailing list.
 # These patchsets are ordered by their recieved time in the lore.kernel.org servers.
 function show_latest_patchsets_from_mailing_list()
 {
-  local -a new_patches
-  local fallback_message
+  local starting_index
+  local ending_index
+  local box_title
+  local extra_label='Previous'
+
+  create_loading_screen_notification "Loading patchsets from ${current_mailing_list} list (page ${PAGE})"
```

Comment:
I'm unsure if I like the `page ${PAGE}` part... to me, it is a little bit weird that we see "page 1" and after two next, we see "page 3". If we can show something like "pages 1-2", it should be fine. If it is not possible, maybe we should not display any number to avoid misleading users.

Reply:
Hmm, you are right. But I don't know if we can know in advance the number of patchsets fetched, as this number fluctuates across queries. Think the best approach for now is to not display the number.